tests/BLE/test_ble.cpp

```diff
@@ -468,6 +478,16 @@ const char* test_mfgdata[][3] = {
     {"BM2", "Li Battery Monitor", "4c000215655f83caae16a10a702e31f30d58dd82f441423149"},
     {"H5104", "GVH5104_1234", "0100010103f99e64"},
     {"H5179", "Govee_H5179_1234", "0188ec000101ee07581641"},
+    {"Oral-B", "Oral-B", "dc0004710502360000000f0004"},
+    {"Oral-B", "Oral-B", "dc000471050332010301030a04"},
+    {"Oral-B", "Oral-B", "dc00047105013a000002010004"},
+    {"Oral-B", "Oral-B", "dc000471050432010e03032e04"},
+    {"Oral-B", "Oral-B", "dc000471057332010e05032e04"},
+    {"Oral-B", "Oral-B", "dc000471050332010e06032e04"},
+    {"Oral-B", "Oral-B", "dc000471050332010e07032e04"},
+    {"Oral-B", "Oral-B", "dc000471050332010e08032e04"},
+    {"Oral-B", "Oral-B", "dc000202060220000001010004"},
+    {"Oral-B", "Oral-B", "dc000202067320020f07080004"},
```

Comment:
And here too

Reply:
These are fine though as they only specify the test name and device braodcast name, with the latter not relevant for this decoder.